logger/logger_test.go

```diff
@@ -100,6 +101,7 @@ func TestLogger_LogWithOptions(t *testing.T) {
 			App:       "app",
 			Env:       "development",
 			Level:     "info",
+			Timestamp: time.Now(),
```

Comment:
just pull this out into a variable, then we can check it strictly later without `InDelta` 

```golang
now := time.Now()

o := Options{
  ...
  Timestamp: now
}

assert.Equal(t, now.UnixNano()/int64(time.Millisecond), int64(line["timestamp"].(float64)))
```

(untested but works maybe)